public/styles/components/_datepicker.css

```diff
@@ -0,0 +1,744 @@
+.CalendarDay {
+  border: 1px solid #e4e7e7;
+  padding: 0;
+  box-sizing: border-box;
+  color: #565a5c;
```

Comment:
The colour values should really be pulled out into the _variables.scss file rather than hardcoding them all.